src/main/java/name/valery1707/problem/ParseJsonList.java

```diff
@@ -82,13 +73,10 @@ public void setCode(String code) {
     }
 
     static String generate(ObjectMapper mapper, int count) throws JsonProcessingException {
-        List<DemoDto> list = IntStream
-            .generate(() -> ThreadLocalRandom.current().nextInt(count * 1000))
-            .distinct()
-            .limit(count)
-            .mapToObj(String::valueOf)
-            .map(DemoDto::new)
-            .collect(toList());
+        List<DemoDto> list = filler(DemoDto::new)
+            .setter(DemoDto::setCode, ints().bound(count * 1000).map(String::valueOf))
```

Comment:
The value of `code` will be used as the key for the map and it should be distinct